changelog/590.breaking.rst

```diff
@@ -0,0 +1 @@
+548.breaking.rst
```

Comment:
I'm not really seeing how this change is breaking - on the surface it essentially just removes the requirement of `entity_type` if a corresponding `channel` is given instead, doesn't it?
cc @onerandomusername

Reply:
This is a symlink to 548's change note, which was breaking, and these changes were added into that changefragment. See [my comment above](https://github.com/DisnakeDev/disnake/pull/617#discussion_r924086077) as to why it is like this.